src/server/resolve-options.js

```diff
@@ -0,0 +1,39 @@
+export function resolveOptions (options, done) {
```

Comment:
Only other way I could invision, is to put the whole render on a setTimeout. This will allow any asynchronous flow to be resolved. Would also be more friendly to vue-router ect. With much less code, will mean `renderToString` and `renderToStream` need to output result to a callback.